llvm/utils/TableGen/CodeGenInstruction.h

```diff
@@ -17,6 +17,7 @@
 #include "llvm/ADT/StringMap.h"
 #include "llvm/ADT/StringRef.h"
 #include "llvm/CodeGenTypes/MachineValueType.h"
+#include "llvm/TableGen/Record.h"
```

Comment:
Done.

Thank you for the review.

Reply:
Or we can move implementation of `isVariableLengthEncoding` to `llvm/utils/TableGen/CodeGenInstruction.cpp` so that we don't need to mess includes. :-)